app/controllers/locations/service_requests_controller.rb

```diff
@@ -3,6 +3,7 @@ class Locations::ServiceRequestsController < ApplicationController
   before_action :authenticate_customer!
   
   def new
+    redirect_to profile_path, alert: 'Complete Registration.' unless current_customer.active
```

Comment:
I think it's better to move it to before_action. Why it's only in this action? Maybe need to add it to another actions too?